test cases/common/223 custom target input extracted objects/verify.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+
+import sys, os
```

Comment:
__[Flake8]__ [F401] 'os' imported but unused ([link](https://sider.review/gh/repos/19784232/pulls/5695#issue-4431737))

<sub>posted by Sider</sub>

Reply:
Fixed in e5093a7c